app/webpacker/components/SearchWidget/MultiSearchInput.jsx

```diff
@@ -20,31 +21,45 @@ const classToComponent = {
   incident: IncidentItem,
 };
 
-function ItemFor({ item }) {
-  const Component = classToComponent[item.class];
-  return (
-    <div className="selected-item">
-      <Component item={item} />
-    </div>
-  );
+function ItemFor({ item, model }) {
+  switch (model || '') {
```

Comment:
Veto for this entire switch block. `classToComponent` already does what you need. The component returned by that lookup can do whatever it wants with the data, like wrap it in a `UserItem` as you are doing here.

Reply:
Okay changed.